web-common/src/app.css

```diff
@@ -115,6 +115,35 @@
 @layer base {
   :root {
     --ui-editor-error-foreground: hsl(0, 75%, 45%);
+
+    /* Defaults for the theme. This will be overridden based on the selected theme.
+       Should match src/features/themes/color-config.ts */
+    /* Primary colors */
+    --color-primary-50: #eff6ff;
+    --color-primary-100: #dbeafe;
+    --color-primary-200: #bfdbfe;
+    --color-primary-300: #93c5fd;
+    --color-primary-400: #60a5fa;
+    --color-primary-500: #3b82f6;
+    --color-primary-600: #2563eb;
+    --color-primary-700: #1d4ed8;
+    --color-primary-800: #1e40af;
+    --color-primary-900: #1e3a8a;
+    --color-primary-950: #172554;
+
+    --color-primary-graph-line: 224;
+    --color-primary-graph-area: 217;
+
+    --color-primary-graph-scrubbing-line: 217;
+    --color-primary-graph-scrubbing-area: 225;
+    --color-primary-scrub-box: 213;
+    --color-primary-scrub-area-0: 221;
+    --color-primary-scrub-area-1: 221;
+    --color-primary-scrub-area-2: 221;
+
+    /* Secondary Colors */
+    --color-secondary-gradient-max: 300;
+    --color-secondary-gradient-min: 1;
```

Comment:
It'd be helpful for the variable names to specify that these numbers are hue numbers, e.g. `--color-primary-graph-line-hue`